packages/shared/src/modules/users/libs/enums/user-validation-regex-rule.enum.ts

```diff
@@ -0,0 +1,15 @@
+import { UserValidationRule } from "./user-validation-rule.enum.js";
+
+const UserValidationRegexRule = {
+	EMAIL_VALID_CHARS_MIN_MAX: new RegExp(
```

Comment:
Let's make sure these regex conditions are aligner with QA team so that they will not forget to test it